File: PRSWeb/Models/PurchaseRequestLineItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;

namespace PRSWeb.Models {
	public class PurchaseRequestLineItem {
		public int Id { get; set; }

		[Required]
		public int Quantity { get; set; }
		
		public int PurchaseRequestId { get; set; }
		public PurchaseRequest PurchaseRequest { get; set; }

		public int ProductId { get; set; }
		public virtual Product Product { get; set; }
	}
}

File: ng/src/app/models/PurchaseRequestLineItem.ts
import { PurchaseRequest } from './PurchaseRequest';

export class PurchaseRequestLineItem {
	
	Id: number;
	ProductId: number;
	PurchaseRequest: PurchaseRequest;
	//PurchaseRequestId: number;
	Quantity: number;

	constructor(
		Id: number,
		ProductId: number,
		PurchaseRequest: PurchaseRequest,
		//PurchaseRequestId: number,
		Quantity: number
	) {
		this.Id = Id;
		this.ProductId = ProductId;
		this.PurchaseRequest = PurchaseRequest;
		//this.PurchaseRequestId = PurchaseRequestId;
		this.Quantity = Quantity;
	}
	
}